app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We updated the search to allow users to find applications by reference in addition to candidate name.
+
+The search works independently from the filters, allowing users to filter or search in whatever way suits their needs.
```

Comment:
How does splitting out ‘allow users to filter or search in whatever way suits their needs’ in comparison to the previous design?

Reply:
In the previous design, the search was incorporated into the filters. If a user cleared all filters they also cleared the search. This may or may not be desirable.

This whole sentence is a duplicate of one in the how it works section, so I have removed it in favour of the latter sentence.